Yoast/yoast-tailwind-columns-block
# Setup Tailwind CSS for the column block

# Summary

This PR adds Tailwind CSS as a Post CSS plugin (see https://tailwindcss.com/docs/installation/using-postcss). 

The Tailwind configuration is the default one in Yoast admin ui toolkit (https://github.com/Yoast/admin-ui/blob/trunk/packages/toolkit/configs/tailwind.js) with a few changes.

In package.json, two new commands are added `build:tailwind` and `watch:tailwind` that scan all the JavaScript files in the `/src` folder and generate CSS code based on the Tailwind utilities used in these files.

`build:tailwind` outputs a minified and production optimized `style.min.css` file in the `/build` folder. 

`watch:tailwind` watch the `/src` folder for any change, outputs and updates a `style.css` file in the `/build` folder.


# How to test
- After running `npm install`, add a Tailwind utility class to one of the JavaScript file in `/src`. For example, update https://github.com/Yoast/yoast-tailwind-columns-block/blob/c18c9adca114d926d2b7e7b73017de53751512ab/column/src/edit.js#L103-L105 to:

```
<div { ...blockProps } class="yst-bg-indigo-100">
    <InnerBlocks />
</div>
```
- Run `npm run build:tailwind` 

There should be a new `style.min.css` file in the build folder.
